shared/product.go

```diff
@@ -78,10 +79,13 @@ func ManageService(product, action, nodeType string, ips []string) (string, erro
 		if getError != nil {
 			return ip, getError
 		}
-		_, err := RunCommandOnNode(cmd, ip)
+		manageServiceOut, err := RunCommandOnNode(cmd, ip)
 		if err != nil {
 			return ip, err
 		}
+		if manageServiceOut != "" {
```

Comment:
this debug stuff you deleted from the other thread but added here on the same situation?

Reply:
you wanted to remove the stdout for RunCommand Logging for security purposes. Your recommendation was to add better verbage per the action. And hence am adding during the actual action for context and debugging purposes.